drivers/docker/driver.go

```diff
@@ -835,6 +835,12 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		return c, fmt.Errorf("requested runtime %q is not allowed", containerRuntime)
 	}
 
+	// Only windows supports alternative isolations modes
+	isolationMode := driverConfig.Isolation
+	if runtime.GOOS != "windows" && driverConfig.Isolation != "process" {
+		return c, fmt.Errorf("Failed to create container configuration for image %q, cannot use isolation mode \"%v\" on %v", driverConfig.Image, isolationMode, runtime.GOOS)
```

Comment:
Looking at the other errors in this area, I'm pretty sure the caller wraps this in a top-level error message as it bubbles up. We probably don't need the image name here, either, because this will get bubbled up to a specific taskrunner's logger and the TaskEvent (which is specific to the task too). Also, we know these are strings so we can use `%s` (or `%q` for quoted strings without manually writing the escaping).

```suggestion
		return c, fmt.Errorf("cannot use isolation mode %q on %s", isolationMode, runtime.GOOS)
```

Reply:
I see that now and have pushed those changes up.